tests/integration/utils2.go

```diff
@@ -1670,10 +1718,11 @@ func assertRequestResults(
 	result *client.GQLResult,
 	expectedResults []map[string]any,
 	expectedError string,
+	asserter ResultAsserter,
 	nodeID int,
 	anyOfByField map[docFieldKey][]any,
 ) bool {
-	if AssertErrors(s.t, s.testCase.Description, result.Errors, expectedError) {
+	if s.isBench || AssertErrors(s.t, s.testCase.Description, result.Errors, expectedError) {
```

Comment:
question: Why `s.isBench` here?

Reply:
to skip assertion. If you run benchmark you don't bother asserting the query result, hence the expected (not set) value would be `nil`

Reply:
> to skip assertion. If you run benchmark you don't bother asserting the query result, hence the expected (not set) value would be `nil`

might be worth it to document that here.

Reply:
You might want to benchmark an error path as well. Anyway I don't think asserting anything but performance is the job of `BenchmarkAction`

Reply:
> Anyway I don't think asserting anything but performance is the job of BenchmarkAction

No, but it is always nice to know when you have messed up and broken the benchmark and caused it to error out instead of doing what you want it to.

If you never check that it has errored you will never know unless you happen to be debugging it (or you notice that performance is now 50 times better etc :)).

Reply:
if we assume that we will never benchmark an error path, then it makes sense to check error here.

Reply:
> if we assume that we will never benchmark an error path, then it makes sense to check error here.

We might want to at somepoint, the bench action should probably allow this then.  Such a feature of the action could be added when the need arises though, not now.  I do think it is important to know right now if the benches stop working though.

Reply:
okay, changed